bagua/torch_api/communication.py

```diff
@@ -134,6 +142,21 @@ def get_intra_node_communicator(self) -> B.BaguaSingleCommunicatorPy:
         return get_communicator(self.group_name, "intra")
 
 
+@lru_cache(maxsize=None)
+def get_rank_mappings():
```

Comment:
global barrier, do I need to pay special attention to using this function? suggest to mark as internal use function